app/kong-ingress-controller/1.0.x/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Documentation for Kong Ingress Controller
+is_homepage: true
+skip_read_time: true
+---
+<div class="docs-grid">
+
+  <h2 class="docs-grid-section-title">Foundations</h2>
```

Comment:
Might be bikeshedding, but I think that we should feature "Guides" here as well. Consider this optional.

Reply:
Might as well. 

Actually - if you take a look at the Enterprise landing page (https://docs.konghq.com/enterprise/), there's a section with a couple of the most popular guides and references (or, things that we just want to call out, really). Could you identify 2-4 of the most useful/popular docs that I could link in a section like that?

Reply:
IMO using-consumer-credential-resource.md and using-kongplugin-resource.md for sure--they're basic tasks that almost everyone will perform. If we want more, setting-up-custom-plugins.md (fairly common) and cert-manager.md (useful and we get a fair number of questions related to it).

Reply:
So - this topic is now gone, as Shane and co don't want KIC at the top level as its own "product". However, I can still carve out a space on the introduction and the guides landing page for some kind of spotlight for important guides.